x-pack/plugin/esql/compute/src/test/java/org/elasticsearch/compute/CountingCircuitBreaker.java

```diff
@@ -0,0 +1,69 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+package org.elasticsearch.compute;
+
+import org.elasticsearch.common.breaker.CircuitBreaker;
+
+public class CountingCircuitBreaker implements CircuitBreaker {
```

Comment:
I'd use `MockBigArrays.LimitedBreaker` with a limit of `1GB` or something. At least, that's what I've been doing and it seemed ok. Maybe it's overkill. But I needed it with my CONCAT PR.

Reply:
That sounds good. I'll take a look

Reply:
I used LimitBreaker of 1Gb, and removed CountCircuitBreaker.